Libraries/Storage/AsyncStorage.js

```diff
@@ -105,6 +105,22 @@ var AsyncStorage = {
   /**
    * Merges existing value with input value, assuming they are stringified json.
    * Returns a `Promise` object. Not supported by all native implementations.
+   *
+   * Example:
+   * ```JavaScript
```

Comment:
lower case javascript to be compatible with github (the current implementation only supports js so it throws it away fyi)